config/locales/docs/sk.yml

```diff
@@ -1503,6 +1503,241 @@ sk:
               description: Mesto
             <<: *timestamp_columns
 
+    ra:
+      change_columns: &change_columns
+        id:
+          description: Identifikátor záznamu
+          note_html: Interný systémový identifikátor. Nepochádza zo zdroja.
+        change_id:
+          description: Identifikátor zmeny
+        changed_at:
+          description: Dátum a čas vykonania zmeny v Registri adries
+        database_operation:
+          description: Vykonaná databázová operácia
+        version_id:
+          description: Identifikátor verzie záznamu
+        created_reason:
+          description: Dôvod vytvorenia záznamu
+        valid_from:
+          description: Dátum a čas platnosti od
+          note_html: Hraničný dátum platnosti o 1.1.1000 má hodnotu <code>NULL</code>
+        valid_to:
+          description: Dátum a čas platnosti do
+          note_html: Hraničný dátum platnosti do 31.12.3000 má hodnotu <code>NULL</code>
```

Comment:
Nie, je to prebrane z original dokumentacie (napr [tu](https://data.gov.sk/dataset/f2b3af7a-b4d4-45d7-9605-29fa5d7ad115/resource/29936968-f4c5-4afc-8c71-a2da2ae142bf/download/package6doc1.htm)). Chapem tomu tak, ze ak je hodnota `NULL`, znamena to datum platnosti do 31.12.3000.

Preformulujem na `Hodnota NULL znamená dátum platnosti do 31.12.3000`.